File: FormulaOneManagementSimulator/Models/Ratings/CarRating.cs
public class CarRating : ICarRating
{
    public CarRating
    (
       uint[] ratings
    )
    {
        Aero = ratings[0];
        Engine = ratings[1];
        Reliability = ratings[2];
        TireManagement = ratings[3];
        CalculateOverall();
    }

    public uint Aero { get; }
    public uint Engine { get; }
    public uint Reliability { get; }
    public uint TireManagement { get; }
    public uint Overall { get; private set; }

    private void CalculateOverall()
    {
        Overall = (Aero + Engine + Reliability + TireManagement) / 4;
    }
}

File: src/models/ratings/car_rating.rs
#[derive(Debug, Copy, Clone, PartialEq, Eq)]
pub struct CarRating {
    pub aero: u32,
    pub engine: u32,
    pub reliability: u32,
    pub tire_management: u32,
    pub overall: u32,
}
impl CarRating {
    pub fn new(seeds: [u64; 5]) -> CarRating {
        todo!()
    }
}
